python/run-tests.py

```diff
@@ -53,20 +53,30 @@ def delete_if_exists(path):
         print("Deleted %s " % path)
 
 
-def prepare(root_dir):
+def prepare(root_dir, use_spark_master):
     print("##### Preparing python tests & building packages #####")
     # Build package with python files in it
     sbt_path = path.join(root_dir, path.join("build", "sbt"))
     delete_if_exists(os.path.expanduser("~/.ivy2/cache/io.delta"))
     delete_if_exists(os.path.expanduser("~/.m2/repository/io/delta/"))
-    run_cmd([sbt_path, "clean", "publishM2"], stream_output=True)
+    sbt_command = [sbt_path]
+    packages = ["spark/publishM2", "storage/publishM2"]
+    if use_spark_master:
+        sbt_command = sbt_command + ["-DsparkVersion=master"]
+        packages = packages + ["connectCommon/publishM2", "connectServer/publishM2"]
+    run_cmd(sbt_command + ["clean"] + packages, stream_output=True)
 
+
+def get_local_package(package_name, use_spark_master):
```

Comment:
This used to run delta connect tests in https://github.com/delta-io/delta/commit/5c608fa108dacffe47cfb93ea9eeeda5080ea357